homeassistant/components/victron_ble/custom_state_data.py

```diff
@@ -0,0 +1,44 @@
+"""Custom state data for victron-ble. See https://github.com/Bluetooth-Devices/sensor-state-data/pull/47."""
```

Comment:
This module should not be in Home Assistant.

Reply:
Got it, I have it following the `VictronBluetoothDeviceData` class to the victron-ble dependency.